x-pack/plugins/observability_solution/investigate_app/public/pages/details/components/investigation_details/investigation_details.tsx

```diff
@@ -5,33 +5,25 @@
  * 2.0.
  */
 
-import { EuiFlexGroup, EuiFlexItem, EuiLoadingSpinner } from '@elastic/eui';
+import { EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
 import { AuthenticatedUser } from '@kbn/security-plugin/common';
 import React from 'react';
-import { useFetchInvestigation } from '../../../../hooks/use_fetch_investigation';
 import { InvestigationItems } from '../investigation_items/investigation_items';
 import { InvestigationNotes } from '../investigation_notes/investigation_notes';
 
 interface Props {
   user: AuthenticatedUser;
-  investigationId: string;
 }
 
-export function InvestigationDetails({ user, investigationId }: Props) {
-  const { data: investigation, isLoading } = useFetchInvestigation({ id: investigationId });
-
-  if (isLoading || !investigation) {
```

Comment:
With the new refactored code how could we add a spinner when it's still loading or if there's no investigation? Or I guess this logic is moved to the parent, right?

Reply:
Yes, I actually added some spinners in the PR I'm working on for the edit form.